integration/test-integration/src/main/resources/META-INF/services/org.wso2.carbon.apimgt.common.gateway.jwttransformer.JWTTransformer

```diff
@@ -0,0 +1 @@
+org.wso2.choreo.connect.tests.CustomJWTTransformer
```

Comment:
Do we need a jar to be committed in the repo

Reply:
not committed any where. Rather user needs to add it to dropins directory.
When creating the custom impl jar, the user needs to mention his class under  `META-INF/services/org.wso2.carbon.apimgt.common.gateway.jwttransformer.JWTTransformer` file